docs/concepts/runtime.md

```diff
@@ -74,10 +74,10 @@ To control how many tasks a worker accepts, a worker has so called **task slots*
 Each *task slot* represents a fixed subset of resources of the TaskManager. A TaskManager with three slots, for example,
 will dedicate 1/3 of its managed memory to each slot. Slotting the resources means that a subtask will not
 compete with subtasks from other jobs for managed memory, but instead has a certain amount of reserved
-managed memory. Note that no CPU isolation happens here; currently slots only separate the managed memory of tasks.
+managed memory. No CPU isolation happens, slots only separate the managed memory of tasks.
```

Comment:
"," -> ";"?

Reply:
@greghogan With the full stop I don't think it's needed anymore.